src/main/java/com/uber/cadence/testing/TestWorkflowEnvironment.java

```diff
@@ -27,13 +28,13 @@
 /**
  * TestWorkflowEnvironment provides workflow unit testing capabilities.
  *
- * <p>Testing workflow code is hard as it might be potentially very long running. The included
- * in-memory implementation of the Cadence service supports <b>automatic time skipping</b>. Anytime
- * a workflow under the test as well as a unit test code are waiting on a timer (or sleep) the
- * internal service time is automatically advanced to the nearest time that unblocks one of the
- * waiting threads. This way a workflow that runs in production for months is unit tested in
+ * <p>Testing the workflow code is hard as it might be potentially very long running. The included
+ * in-memory implementation of the Cadence service supports <b>an automatic time skipping</b>.
+ * Anytime a workflow under the test as well as the unit test code are waiting on a timer (or sleep)
+ * the internal service time is automatically advanced to the nearest time that unblocks one of the
+ * waiting threads. This way a workflow that runs in the production for months is unit tested in
```

Comment:
runs in the production -> runs in production OR runs in the production environment